libs/commons/dune

```diff
@@ -25,6 +25,7 @@
      ppx_deriving.show
      ppx_deriving.eq
      ppx_deriving.ord
```

Comment:
see we already include ppx_deriving.ord.
Did you try deriving ord instead? What was wrong with it?